tests/test_integration/test_pipeline.py

```diff
@@ -127,19 +129,18 @@ def test_linear():
     }
     fit = sampling.sample(**linear_input_values)
     samples_test = fit.draws_pd()
-    dict(
-        zip(
-            samples_test.columns,
-            map(list, samples_test.quantile([0.03, 0.97]).T.values),
-        )
+    expected_df = pd.DataFrame.from_dict(
+        expected, orient="index", columns=["5% CI", "95% CI"]
     )
-    assert list(samples_test.columns) == list(expected.keys())
```

Comment:
Rather than deleting this check could we perhaps change it to the following?

```
assert all([k in samples_test.columns for k in expected.keys()])
```